sources/packages/web/src/views/student/StudentApplicationDetails.vue

```diff
@@ -57,7 +57,7 @@
     <student-assessment-details :applicationId="id" v-if="showViewAssessment" />
   </student-page-container>
 
-  <confirm-edit-application ref="editApplicationModal" />
+  <confirm-edit-application ref="editApplicationModal" :beforeEdit="true" />
```

Comment:
I know many of our old code is not following the hyphen separated format for Vue components and it's attributes. 

![image](https://github.com/user-attachments/assets/24f94061-dcf4-4257-9962-9dd2bec60d1c)

Reply:
I meant for the new attribute please follow the that pattern `before-edit` instead of `beforeEdit`

Reply:
updated